CHANGELOG.md

```diff
@@ -6,19 +6,50 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```

Comment:
If this file still follows the https://keepachangelog.com/en/1.0.0/ "standard" there should be a running "Unreleased" entry which is added to between releases.

Reply:
Done.